Dockerfile

```diff
@@ -15,6 +15,7 @@ echo "**** install packages ****" && \
  apk add --no-cache --upgrade && \
  apk add --no-cache \
 	nodejs && \
+	libxslt-dev && \
 echo "**** fetch sickchill ****" && \
```

Comment:
I would do libxslt not libxslt-dev the dev package is strictly for building stuff from source. 
Also this package should be above nodejs (alphabetical order) and does not need a && operator.

Reply:
yeah sure